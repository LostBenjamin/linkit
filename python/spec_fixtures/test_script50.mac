# ------- test 1 -----------
%include test_script50_include1

[testmacro]
  blue

  (testmacro)
  (test_script50_include1-test1macro)
  (test_script50_include2-test2macro)

