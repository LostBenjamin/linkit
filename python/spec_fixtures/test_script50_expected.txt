4:set:blu:4:run:5:run:6:run
5:set:grn:5:run:6:run
6:set:red:6:run

